### ISSUE TITLE
How to get styles of focused text or content

### ISSUE THREAD

**Issue by JMA12** [facebook/draft-js#2109]
How can I get the inline styles that focused content have?

**Comment by jobpaardekooper** [comment]
You can get an array of the current inline styles by doing the following.
‘’’
const currentFocus = editorState.getSelection().getFocusKey();

const inlineStyles = editorState.getCurrentInlineStyle(currentFocus);
‘’’

As also descrived here: https://github.com/facebook/draft-js/issues/602#issuecomment-503209917

**Comment by jkhaui** [comment]
@jobpaardekooper Just a heads up, you don't pass any arguments to `getCurrentInlineStyle()` I'm pretty sure. So just using `const inlineStyles = editorState.getCurrentInlineStyle();` on its own is enough to return an OrderedMap which holds the inline styling info

**Comment by jobpaardekooper** [comment]
@jkhaui you are correct according to [the documentation](https://draftjs.org/docs/api-reference-editor-state#getcurrentinlinestyle).
